source/javascripts/services/request-service.ts

```diff
@@ -97,12 +91,7 @@ class RequestService {
 			throw error;
 		}
 
-		try {
-			return await response.text();
-		} catch (error) {
-			this.logErrorWithLevel(error, StatusType.error);
-			throw new Error(window["strings"].request_service.load_app_config.default_error);
-		}
+		return await this.convertResponseToText(response, window["strings"].request_service.load_app_config.default_error);
```

Comment:
this await is not harmful here, but it doesn't do anything here, you can just simply return the promise here